api/src/main/java/org/pmiops/workbench/api/AuthDomainController.java

```diff
@@ -38,11 +44,17 @@
   @Override
   @AuthorityRequired({Authority.ACCESS_CONTROL_ADMIN})
   public ResponseEntity<Void> updateUserDisabledStatus(UpdateUserDisabledRequest request) {
-    DbUser user = userDao.findUserByEmail(request.getEmail());
-    Boolean previousDisabled = user.getDisabled();
-    DbUser updatedUser = userService.setDisabledStatus(user.getUserId(), request.getDisabled());
+    final DbUser targetDbUser = userDao.findUserByEmail(request.getEmail());
+    final Boolean previousDisabled = targetDbUser.getDisabled();
+    final DbUser updatedTargetUser =
+        userService.setDisabledStatus(targetDbUser.getUserId(), request.getDisabled());
+    authDomainAuditAdapter.fireSetAccountEnabled(
```

Comment:
Is there a plan to reconcile this? I don't like this middle-ground where most of the codebase talks about "disabling" users, and then one part of the code switches to talking about "enabling" them. They are not conceptually exact inverses and I think there could have been a renaming or switch to an enum could have improved things instead without the inconsistency.

Reply:
I just thought it was a best practice to have any enum be "active high", and be named after a positive rather than a negative.

Is it worth doing a database migration and backfill to get the whole code base consistent? Definitely a tough sell. It's the classic tension between consistency and incremental improvement.

On the other hand, the auditing schema is flexible here, and I can certainly use an enum on my side. If you feel strongly that consistency trumps clarity (as measured by number of mental inversions to interpret the variable), then it's easy enough to take in an `isDisabled` and `previousWasDisabled` value, and rename the function.

Reply:
I'll buy your argument about the signature here matching the existing sense of the disabled columns and fields. I'm still thinking bout the best way to represent it in the audit schema, but I think your idea of an enum instead of a boolean fits well here.

The conversation around disabled & enabled not being 100% opposites makes me think we shouldn't be using booleans at all for this state on the DB side.